iwf/internal_mapper.go

```diff
@@ -52,23 +52,24 @@ func toIdlCommandRequest(commandRequest *CommandRequest) (*iwfidl.CommandRequest
 	var signalCmds []iwfidl.SignalCommand
 	var interStateCmds []iwfidl.InterStateChannelCommand
 	for _, t := range commandRequest.Commands {
+		commandId := t.CommandId
 		if t.CommandType == CommandTypeTimer {
 			timerCmd := iwfidl.TimerCommand{
-				CommandId:                  t.CommandId,
-				FiringUnixTimestampSeconds: t.TimerCommand.FiringUnixTimestampSeconds,
+				CommandId:       &commandId,
+				DurationSeconds: t.TimerCommand.FiringUnixTimestampSeconds,
```

Comment:
Actually this is different. see https://github.com/indeedeng/iwf-java-sdk/pull/229 and cc @samuel27m who also has the context

Reply:
@longquanzheng I added the change to replace `FiringUnixTimestampSeconds` with `DurationSeconds` in this MR since it was dependent on the sdk update. Tests all pass now.